src/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  flex-basis: 300px;
+  flex-grow: 1;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
move `font-family` to `html`

Reply:
error in HTML ~The tag <style> is banned and should not be used~

Reply:
I meant selector by tag html)

Reply:
done